Modules/Video/Core/wrapping/itkImageToVideoFilter.wrap

```diff
@@ -0,0 +1,20 @@
+itk_wrap_include("itkImage.h")
+
+# Do not wrap for input of lowest dimension D as this would result in
+# output frames of dimension D-1 which are not supported.
+# Here we assume that either 1 or 2 is the lowest image dimension wrapped.
```

Comment:
Is this a reasonable assumption? An alternative is sorting and removing the last item in the list, but without natural sort (CMake 3.18) that opens up an edge case where the user has wrapped for 10+ dimensional images. Not sure how likely this is compared to not wrapping for at least dimension 2.

Reply:
Rather than trying to omit the lowest listed dimension in `${ITK_WRAP_IMAGE_DIMS}`, possibly augmented, does it make sense to instead use all the listed dimensions, with 1 added to each?
```cmake
foreach(d ${ITK_WRAP_IMAGE_DIMS})
  # Use d+1 in wrappings
endforeach()

```

Reply:
I think we still end up with the same problem of what to exclude. If we support `Image`s of dimensions M,...,N and try to wrap the filter for `Image` inputs of dimensions M+1,...,N+1 we still need a way to sort and remove the unsupported N+1 dimension.

Given that `ImageToVideoFilter` reduces the dimensionality of `Image` objects in use by 1 (i.e. `Image<t,d>` -> `VideoStream<Image<t,d-1>>`, etc), we're going to have to drop either the lowest or highest image dimension.

Reply:
You don't need a `sort`, right, just a `min`?  Are the .wrap files parsed by CMake?  Something like,

`
set(m_d "-1")
foreach(d ${ITK_WRAP_IMAGE_DIMS})
  if  ($m_d LESS $D)
     set(m_d "$d")
  endif()
endforeach()
string(REPLACE $m_d "" NEW_WRAP_IMAGE_DIMS "${ITK_WRAP_IMAGE_DIMS}")
`

Reply:
Looks like this is the right answer, thanks @brad-t-moore! Was unclear on value comparisons in CMake. Wrapping file is amended.